builds/misc/packages-release.yaml

```
@@ -435,5 +435,161 @@ stages:
           env:
-            GITHUB_PAT: "$(GitHubAccessToken)"    
-          name: publish_artifacts 
+            GITHUB_PAT: "$(GitHubAccessToken)"
+          name: publish_artifacts
+
+##############################################################################
+  - stage: smokeTests
+##############################################################################
+    displayName: Post-release Smoke Tests
+    dependsOn:
+      - PublishPackagesMicrosoft
+      - PublishPackagesGithub
+    jobs:
+    - job: smokeTests
+      displayName: Release Artifact Smoke Tests
+      strategy:
+        matrix:
+          Ubuntu1804-amd64:
+            pool_name: 'Azure-IoT-Edge-1ES-Hosted-Linux'
+            agent_demands: 'ImageOverride -equals agent-aziotedge-ubuntu-18.04-docker'
+            os: ubuntu18.04
+            arch: amd64
+            ext: deb
+          Ubuntu1804-arm64:
+            pool_name: 'Azure-IoT-Edge-1ES-Hosted-Linux-Arm64'
+            agent_demands: 'ImageOverride -equals agent-aziotedge-ubuntu-18.04-arm64'
+            os: ubuntu18.04
+            arch: arm64
+            ext: deb
+
+      pool:
+        name: $(pool_name)
+        demands:
+        - $(agent_demands)
+
+      steps:
+      - checkout: azure-iotedge
+        submodules: recursive
+      - checkout: self
+      - task: AzureKeyVault@1
+        displayName: Get secrets
+        inputs:
+          azureSubscription: $(az.subscription)
+          keyVaultName: $(kv.name)
+          secretsFilter: >-
+            IotEdge1-PAT-msazure
+      - task: Bash@3
+        displayName: Setup Test Agent
+        condition: true
+        name: SetupTestAgent
+        inputs:
+          targetType: 'inline'
+          script: |
+            # Source the scripts & Update version files
+            sudo chmod +x $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
+            source $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
+
+            wait-for-dpkg-lock 120
+
+            if ! command -v jq &> /dev/null
+            then
+                sudo apt-get install jq -y
+            fi
+
+      - task: Bash@3
+        displayName: Get Parameters From Azure-iotedge
+        condition: true
+        name: GitHubParameters
+        inputs:
+          targetType: 'inline'
+          script: |
+            # Source the scripts and get versions
+            edgeletVersion=$(cat $(Build.SourcesDirectory)/azure-iotedge/latest-aziot-edge.json | jq ".\"aziot-edge\"" | tr -d '"')
+            iisVersion=$(cat cat $(Build.SourcesDirectory)/azure-iotedge/latest-aziot-identity-service.json | jq ".\"aziot-identity-service\"" | tr -d '"')
+
+            echo "##vso[task.setvariable variable=edgeletVersion;isOutput=true]$edgeletVersion"
+            echo "##vso[task.setvariable variable=iisVersion;isOutput=true]$iisVersion"
+        env:
+          GITHUB_PAT: "$(GitHubAccessToken)"
```

Is there reason why you're using the verbose syntax for the Bash task? Why not use the simpler syntax?

```suggestion
      - bash: |
          # Source the scripts and get versions
          edgeletVersion=$(cat $(Build.SourcesDirectory)/azure-iotedge/latest-aziot-edge.json | jq ".\"aziot-edge\"" | tr -d '"')
          iisVersion=$(cat cat $(Build.SourcesDirectory)/azure-iotedge/latest-aziot-identity-service.json | jq ".\"aziot-identity-service\"" | tr -d '"')

          echo "##vso[task.setvariable variable=edgeletVersion;isOutput=true]$edgeletVersion"
          echo "##vso[task.setvariable variable=iisVersion;isOutput=true]$iisVersion"
        displayName: Get parameters from Azure-iotedge
        name: GitHubParameters
        env:
          GITHUB_PAT: "$(GitHubAccessToken)"
```

---

I want to be able to set the `name` and `displayName` for the task. This allows me to later refer to it while using the exported variable.

---

See my example above...it has both `name` and `displayName`.